### DISCUSSION TITLE
Missing data in one table

### DISCUSSION THREAD

**Discussion by agruenberg** [Budibase/budibase#8245] [Help]
I made a small app with several tables and all worked quite well until now ;)
There is a table named companies and has a lot of relationships to other tables. Within this tables, there are 131 records. Budibase shows me since today only two record. I cannot estimate why?
Did anyone else had the same experience?

Datagrip result:
![datagrip](https://user-images.githubusercontent.com/19472717/195623820-159635c0-1017-4875-9262-d665a33a461d.png)

Budibase result:
![budibase](https://user-images.githubusercontent.com/19472717/195623803-7388a220-fc6d-424c-8cb7-e06afe8737ff.png)

Database is MySQL.

**Comment by melohagan** [comment]
Hey @agruenberg 

If you open the console (Fn+F12 on Chrome) do you see any errors? And errors in the network tab?

**Reply by agruenberg** [reply]
Hi @melohagan 
No, there is no error in the console and the response of the network tab shows me also only two records.

**Comment by TommasoBernini** [comment]
Hi, @agruenberg I probably have the same problem. 
I have two tables, with a one-to-many relationship between them established in Budibase (table A are sectors and B prices with the sector propriety). 
At the moment not all the rows are linked because some of the elements of table A do not have a correspondence in table B (eg. some sectors have no price). Since not all the rows in table A are linked to the rows in table B, when I create the relationship magically all the rows that do not have a link with the rows of table B disappear and only the actually used price remain, despite the rows of the A table being present in the DB (Postgres). 
Oddly, when I order the rows of table A using the A-Z sorting of a specific column, some of the missing rows pop up. (app version: 2.0.18)

@agruenberg let me know if your problem has the same phenomenology as mine.

**Reply by agruenberg** [reply]
Hi @TommasoBernini 
My companies table has several 1:n relations to other tables. The two shown rows are also linked to other tables. I don't see any pattern, why these both rows are shown. And yes. if I reorder other rows from this table pop up. Every ordered column shows an other result. It's very odd. My App Version is 2.0.30.
My DB System is MySQL.

**Comment by agruenberg** [comment]
@melohagan, @TommasoBernini Little Update ;)
I deleted now ALL in budibase defined relationships for this table and all data appeared magically ;) Afterwards I startet to rebuild each relation. Relation 1-4 where no problem but when I set the 5th relation, the same problem appears. Is it possible, that budibase can only manage a certain amount of relationships per table?

**Reply by TommasoBernini** [reply]
It might be, but sincerely I don't know. I hope that someone manage to find out what the real problem is and fix it

**Reply by melohagan** [reply]
Thanks for the update @agruenberg 

To clarify, what are the relationships between your tables? 

When you say 'relation', is that a row or a column? I.e. is the 5th relation here a fifth row that has been added, or a fifth relationship column?

**Reply by agruenberg** [reply]
Hi, thank you for your Answer. I am talking about relations in sense of database relations with key / foreign key pairs. So I mean relations by columns.

**Reply by melohagan** [reply]
Okay great! Out of curiosity, what would the total number of rows be between all of the 5 related tables? What about the total rows of the 4 tables that did work?

**Reply by agruenberg** [reply]
I am not sure, if I understood your requests, but that are the metrics (table name -> records inside):
- companies -> 113
- costs -> 741
- projects -> 50
- persons -> 4
- forecasts -> 229
- offers -> 0

The companies table has relations 1:n to the tables costs, projects, persons, forecasts and offers.

**Reply by TommasoBernini** [reply]
any news on this?
